### ISSUE TITLE
ACU death nukes not triggering in 3652

### ISSUE THREAD

**Issue by Crotalus** [FAForever/fa#1029]
Probably an initialization issue, they don't seem to have InnerRing / OuterRing set

**Comment by aeoncleanse** [comment]
It's the nuke refactor I think. Need to take a closer look at what I did in Weapon.lua.

**Comment by aeoncleanse** [comment]
Yup, when I get a moment I'll create a new weapon class for NukeDeathWeapons and move those all around. ACUs, Paragon, is there anything else?

**Comment by speed2CZ** [comment]
maybe SACUs

**Comment by Crotalus** [comment]
Yup, SCUs seems broken

```
WARNING: ...us\documents\dev\fa\units\ual0301\ual0301_script.lua(15): access to nonexistent global variable "AIFCommanderDeathWeapon"
```

**Comment by aeoncleanse** [comment]
Hmm, I'll take a look. The SCUs don't use nuke params, so this is a bit odd.

**Comment by Crotalus** [comment]
Nice, I changed them to use DeathNukeWeapon but now they seem to be missing Radius.

**Comment by aeoncleanse** [comment]
Fixed in 1cc016ea898ee2df51cf623f23d158fbf99382f8
